pyvista/utilities/geometric_objects.py

```diff
@@ -446,16 +450,19 @@ def Line(pointa=(-0.5, 0.0, 0.0), pointb=(0.5, 0.0, 0.0), resolution=1):
     >>> mesh.plot(color='k', line_width=10)
 
     """
-    if resolution <= 0:
-        raise ValueError('Resolution must be positive')
-    if np.array(pointa).size != 3:
-        raise TypeError('Point A must be a length three tuple of floats.')
-    if np.array(pointb).size != 3:
-        raise TypeError('Point B must be a length three tuple of floats.')
     src = _vtk.vtkLineSource()
-    src.SetPoint1(*pointa)
-    src.SetPoint2(*pointb)
-    src.SetResolution(resolution)
+    if points is None:
+        if resolution <= 0:
+            raise ValueError('Resolution must be positive')
+        if np.array(pointa).size != 3:
+            raise TypeError('Point A must be a length three tuple of floats.')
+        if np.array(pointb).size != 3:
+            raise TypeError('Point B must be a length three tuple of floats.')
+        src.SetPoint1(*pointa)
+        src.SetPoint2(*pointb)
+        src.SetResolution(resolution)
+    else:
+        src.SetPoints(pyvista.vtk_points(points))
```

Comment:
Does it make sense to use this helper to do some dimension and shape checking before this line?

https://github.com/pyvista/pyvista/blob/ef205f48454bb5bef09b2d00e38c9b49ec31eb50/pyvista/utilities/common.py#L10